samples/listFilesPaginate.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function main(bucketName = 'my-bucket') {
+  // [START storage_list_files]
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  async function listFilesPaginated() {
+    const bucket = storage.bucket(bucketName);
+    const [files, queryForPage2] = await bucket.getFiles({autoPaginate: false});
+    console.log('Files:');
+    files.forEach(file => {
+      console.log(file.name);
+    });
+    // Get the next set of results
+    if (queryForPage2 !== null) {
+      const [files, queryForPage3] = await bucket.getFiles(queryForPage2);
+      files.forEach(file => {
+        console.log(file.name);
+      });
+
+      if (queryForPage3 !== null) {
+        // Request the nest set of results...
+      }
```

Comment:
```suggestion
      // If necessary, continue cursoring using "queryForPage3"
```

Reply:
Will try, linter may complain though.

Reply:
> error  'queryForPage3' is assigned a value but never used  no-unused-vars

Reply:
I think I've seen that rule ignored for the sake of samples. I forget how specific the rule was-- is it possible to ignore that rule break with a "ignore" line at the top of the file / beginning of the function (outside the printable area)?

Reply:
Done